NAMESPACE

```diff
@@ -325,6 +326,7 @@ importFrom(ape,has.singles)
 importFrom(ape,is.rooted)
 importFrom(ape,read.tree)
 importFrom(ape,root)
+importFrom(biomformat,observation_metadata)
```

Comment:
biomformat is not dependency of mia. This means that biomformat is not installed automatically. This leads to an error.

Instead, you could add .require_package("biomformat") in the beginning of function and specify biomformat functions like biomformat::observation_metadata()